charts/backstage/ci/ci-values.yaml

```diff
@@ -0,0 +1,5 @@
+backstage:
+  image:
+    registry: ghcr.io
+    repository: janus-idp/redhat-backstage-build
```

Comment:
why aren't we using `https://github.com/backstage/backstage/blob/master/packages/backend/Dockerfile`?

Reply:
Yeah, I know. I just wasn't able to find any already built image upstream... therefore I've chose this one, see this PR note:

> Using janus-idp/redhat-backstage-build as deployment image, since I don't know about any other vanilla image out there... I think we want to change this to an upstream image if possible.

I'd love to use any vanilla upstream image, I just wasn't able to find it so for the time being I deferred to this one (the other option was to use roadie's community image which is quite opinionated, this one is not).

Ideally we'd have an upstream image that is built in CI using the latest Backstage. That would be the best test image for this chart. I was just lazy to built one at this point.

Reply:
It's `ghcr.io/backstage/backstage`, please use that one instead. Other options are to use a dummy image or to make it required. Users of this chart will need to supply their own image in the end anyway.